### PR TITLE
docs: add sessionStorage notice

### PR DESCRIPTION
**Pull request by SLdragon**
@zhenyasav , could you help provide the helper link target URL, I will update it in aka service: https://aka.ms/teamsfx-session-storage-notice

Related task: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_sprints/taskboard/AuthAndData/Microsoft%20Teams%20Extensibility/CY21-12.1?workitem=12699312

### CODE REVIEW

**Review by adashen — DISMISSED** [review]

File: packages/sdk/src/credential/teamsUserCredential.browser.ts [hunk 1]
```diff
@@ -236,6 +236,7 @@ export class TeamsUserCredential implements TokenCredential {
 
         const tokenResult: AccessTokenResult = response.data;
         const key = await this.getAccessTokenCacheKey(scopesStr);
+        // Important: tokens are stored in sessionStorage, read more here: https://aka.ms/teamsfx-session-storage-notice
```
Comment (blackchoey) [inline packages/sdk/src/credential/teamsUserCredential.browser.ts:239]: Can we also add this notice to document comments of TeamsUserCredential.getToken() function (as well as other possible places)?  So developers can see this notice when coding, as well as see it in our SDK reference doc.

Comment (SLdragon) [reply packages/sdk/src/credential/teamsUserCredential.browser.ts:239]: Sure, will add it

Comment (SLdragon) [reply packages/sdk/src/credential/teamsUserCredential.browser.ts:241]: Updated

File: packages/sdk/src/credential/teamsUserCredential.browser.ts [hunk 2]
```diff
@@ -132,6 +132,8 @@ export class TeamsUserCredential implements TokenCredential {
   /**
    * Get access token from credential.
    *
+   * Important: Graph access tokens are stored in sessionStorage, read more here: https://aka.ms/teamsfx-session-storage-notice
```
Comment (blackchoey) [inline packages/sdk/src/credential/teamsUserCredential.browser.ts:135]: Maybe no need to mention **Graph** access tokens? as all access tokens besides sso token are stored at session storage.

Comment (SLdragon) [reply packages/sdk/src/credential/teamsUserCredential.browser.ts]: Sure, updated

**Review by blackchoey — APPROVED** [review]

**Pull request merged**